lab/admin-ui-sdk/Lab.md

```diff
@@ -0,0 +1,393 @@
+  - [Step 5: Configure the Adobe Commerce Admin Panel](#step-5-configure-the-adobe-commerce-admin-panel)
+  - [Step 6: Test the integration](#step-6-test-the-integration)
+  - [Step 7: Add the configuration component](#step-7-add-the-configuration-component)
+- [Troubleshooting](#troubleshooting)
+
+## Objective
+
+This lab provides hands-on experience with the Adobe Commerce Admin UI SDK by integrating a single-page application (SPA) into the Admin Panel interface, entirely through JavaScript; without requiring any PHP code.
+
+## Problem statement
+
+The single-page application (SPA) includes a configuration toggle that enables or disables stock validation rules, along with a parameter defining the maximum quantity allowed per item in an order. These settings are securely persisted in App Builder’s lib-files storage and retrieved at runtime to dynamically control webhook execution behavior.
+
+## Helpful resources
+
+- Most of required files are located in the `lab/admin-ui-sdk` folder to streamline setup and avoid time spent on boilerplate creation.
+- [Admin UI SDK documentation](https://developer.adobe.com/commerce/extensibility/admin-ui-sdk/)
+- [Adobe App Builder Extensions documentation](https://developer.adobe.com/app-builder/docs/guides/app_builder_guides/extensions/extensions/)
+
+## Step-by-Step instructions
+
+### Step 1: Configuration files
+
+1. Create the `install.yaml` file
+
+    Place this file at the root of your project. Ensure it correctly targets the `commerce/backend-ui/1` extension point ID to register your application with the Adobe Commerce Admin Panel.
+
+    > A sample file is available in the `lab/admin-ui-sdk` folder.
+
+2. Create the `extension-manifest.json` file
+
+    Also located at the root of your project, this file must define the `platform` as `"web"` and include a unique `id` for your extension. You can append the existing id with your seat number.
+
+    > A sample file is provided in the `lab/admin-ui-sdk` folder.
+
+### Step 2: Migrate from application to extension
+
+1. Create a new `src/commerce-backend-ui-1` folder.
+
+2. Move the `actions` folder under `src/commerce-backend-ui-1`.
+
+3. Move the `web-src` folder under `src/commerce-backend-ui-1`.
+
+4. Move the `utils` folder under `src/commerce-backend-ui-1`.
+
+5. Update the `src/commerce-backend-ui-1/actions/starter-kit-info/index.js` file to reference the correct path for the `version` and `registrations`:
+
+    ```javascript
+    const version = require('../../../../package.json').version
+    const registrations = require('../../../../scripts/onboarding/config/starter-kit-registrations.json')
+    ```
+
+6. Update the `src/commerce-backend-ui-1/utils/naming.js` file to reference the correct path for the `providersList`:
+
+    ```javascript
+    const providersList = require('../../../scripts/onboarding/config/providers.json')
+    ```
```

Comment:
Not totally necessary, but I think it could be helpful to add line numbers where modifications are being made